### PR TITLE
heroku/setep hosted on herko now

### PR DESCRIPTION
**Pull request by Earthii**
https://fbchromeextension.herokuapp.com/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,28 +4,30 @@
   "description": "Chrome Extension for opening facebook chat in current tab. This permits the user to not having to switch tabs.",
   "main": "index.js",
   "dependencies": {
-    "angular": "^1.6.4",
     "@uirouter/angularjs": "^1.0.4",
-  },
-  "devDependencies": {
+    "angular": "^1.6.4",
     "babel-core": "^6.25.0",
```
Comment (ghiobi) [inline package.json]: these are dev dependencies tho

File: package.json [hunk 2]
```diff
@@ -4,28 +4,30 @@
...
     "babel-loader": "^7.1.0",
     "babel-preset-es2015": "^6.24.1",
     "css-loader": "^0.28.4",
     "eslint": "^4.1.1",
     "eslint-loader": "^1.8.0",
+    "express": "^4.15.3",
```
Comment (ghiobi) [inline package.json]: why express?

File: package.json (hunk 1 above)
Comment (Earthii) [inline package.json]: They aren't when they are used server side to host the front end .

File: package.json (hunk 2 above)
Comment (Earthii) [inline package.json]: To host the front end

Comment (Earthii) [inline package.json]: To host the front end

Comment (Earthii) [inline package.json]: To host the front end

Comment (Earthii) [inline package.json]: To host the front end

Comment (Earthii) [inline package.json]: To host the front end

Comment (Earthii) [inline package.json]: What the heck

**Pull request merged**